package.json

```diff
@@ -24,7 +24,7 @@
   "dependencies": {
     "debug": "^4.1.0",
     "json-stringify-safe": "^5.0.1",
-    "lodash": "^4.17.13",
+    "lodash.set": "^4.3.2",
```

Comment:
These mono-function lodash packages are deprecated and have not been updated in over 4 years.

They [should not be used](https://lodash.com/per-method-packages) – and they will be removed in v5.